src/components/__tests__/RouteListEdit.cy.js

```diff
@@ -23,6 +20,10 @@ describe('<RouteListEdit>', () => {
     });
 
     coreTests();
+
+    it('renders items in two columns', () => {
+      cy.testElementPercentageWidth(cy.dataCy('route-list-item-wrapper'), 50);
```

Comment:
Repeatable string or number should be const.

Reply:
Extracted to a variable.